src/sentry/api/endpoints/release_thresholds/release_threshold_details.py

```diff
@@ -41,3 +62,25 @@ def get(
         self, request: Request, project: Project, release_threshold: ReleaseThreshold
     ) -> HttpResponse:
         return Response(serialize(release_threshold, request.user), status=200)
+
+    def put(
+        self, request: Request, project: Project, release_threshold: ReleaseThreshold
+    ) -> HttpResponse:
+        serializer = ReleaseThresholdPUTSerializer(data=request.data)
+
+        if serializer.is_valid():
```

Comment:
early return 😶
```suggestion
        if not serializer.is_valid():
            return Response(serializer.errors, status=400)
```

Reply:
🤷 
time/complexity wise this is the same.